src/main/java/com/github/tomitakussaari/phaas/util/JsonHelper.java

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.experimental.UtilityClass'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)